### ISSUE TITLE
require() of ES modules is not supported.

### ISSUE THREAD

**Issue by xreider** [sindresorhus/crypto-random-string#40]
**GOT ISSUE WITH 4 Version**

```
[1] Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\node_modules\crypto-random-string\index.js
[1] require() of ES modules is not supported.
[1] require() of X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\node_modules\crypto-random-string\index.js from X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\emails\sendingEmails.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
[1] Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\node_modules\crypto-random-string\package.json.
[1]
[1]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1080:13)
[1]     at Module.load (internal/modules/cjs/loader.js:928:32)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[1]     at Module.require (internal/modules/cjs/loader.js:952:19)
[1]     at require (internal/modules/cjs/helpers.js:88:18)
[1]     at Object.<anonymous> (X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\emails\sendingEmails.js:1:16)
[1]     at Module._compile (internal/modules/cjs/loader.js:1063:30)
[1]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[1]     at Module.load (internal/modules/cjs/loader.js:928:32)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[1]     at Module.require (internal/modules/cjs/loader.js:952:19)
[1]     at require (internal/modules/cjs/helpers.js:88:18)
[1]     at Object.<anonymous> (X:\YandexDisk\NodeJSProjects\NODE\HSK.WiKi\HSK.WiKi31\reducers\auth.js:4:30)
[1]     at Module._compile (internal/modules/cjs/loader.js:1063:30)
[1]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[1]     at Module.load (internal/modules/cjs/loader.js:928:32) {
[1]   code: 'ERR_REQUIRE_ESM'
[1] }
[1] [nodemon] app crashed - waiting for file changes before starting...
```

**Comment by sindresorhus** [comment]
Duplicate of #36
